src/css/main.less

```diff
@@ -966,24 +967,33 @@ fieldset[disabled] .btn-info.active {
       padding: @base-spacing-unit*2;
       width: @sidebar-width;
 
-      .btn-success {
+      .btn-success, .dropdown-toggle, .dropdown-menu {
         margin: 0 0 @base-spacing-unit 0;
         text-align: left;
         width: 100%;
       }
 
-      h3 {
-        color: @sidebar-heading-color;
-        font-weight: bold;
-        font-size: @font-size-h3;
-        line-height: @base-spacing-unit*2;
-        text-transform: uppercase;
-        margin: @base-spacing-unit 0;
+      .heading {
+
+        h3 {
```

Comment:
Can we extract this style from the sidebar? Its current selector is `.tab-pane.active.flex-container nav.sidebar .heading h3`. I'd prefer something like `h3.smallcaps`.